Complex backward for torch.sqrt

anjali411 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#49461 Complex backward for torch.sqrt**

resolves https://github.com/pytorch/pytorch/issues/48398

Differential Revision: [D25589454](https://our.internmc.facebook.com/intern/diff/D25589454)

anjali411 committed:
Complex backward for torch.sqrt

[ghstack-poisoned]

mruberry reviewed (approved):
LGTM! Thank you for fixing the eager variant consistency test, too.

anjali411 committed:
Update on "Complex backward for torch.sqrt"


resolves https://github.com/pytorch/pytorch/issues/48398

Differential Revision: [D25589454](https://our.internmc.facebook.com/intern/diff/D25589454)

[ghstack-poisoned]

anjali411 committed:
Update on "Complex backward for torch.sqrt"


resolves https://github.com/pytorch/pytorch/issues/48398

Differential Revision: [D25589454](https://our.internmc.facebook.com/intern/diff/D25589454)

[ghstack-poisoned]

anjali411 committed:
Update on "Complex backward for torch.sqrt"


resolves https://github.com/pytorch/pytorch/issues/48398

Differential Revision: [D25589454](https://our.internmc.facebook.com/intern/diff/D25589454)

[ghstack-poisoned]
